src/validation/json-schemas/index.ts

```diff
@@ -0,0 +1,13 @@
+import defs from './defs.schema.json';
+import GeneralJws from './general-jws.json';
+import PermissionsDefs from './permissions/defs.json';
+import PermissionsRequest from './permissions/permissions-request.json';
+import Request from './request.json';
+
+export const schemas = {
+  defs,
```

Comment:
Consider:
```suggestion
  definitions,
```
and `PermissionDefinitions`

Reply:
done: https://github.com/TBD54566975/dwn-sdk-js/commit/b48197606af323c31f6f31f78e941b4ffc03f61e